apps/abstracts/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+class DateTimeCustom(models.Model):  # noqa
+
+    datetime_created = models.DateTimeField(
+        verbose_name="Время создания",
+        auto_now_add=True
+    )
+    datetime_updated = models.DateTimeField(
+        verbose_name="Время обновления",
+        auto_now=True
+    )
+    datetime_deleted = models.DateTimeField(
+        verbose_name="Время удаления",
+        null=True,
+        blank=True
+    )
+    is_deleted = models.BooleanField(
```

Comment:
do we need "is_deleted" since "datetime_deleted" is presented

Reply:
It was provided in order to obtain an opportunity with list_filter for deleted items

Reply:
Could we use `.filter(datetime_deleted__isnull=True)` for such filtering instead of adding additional boolean field ?